src/ui/interface.js

```diff
@@ -858,28 +859,49 @@ var UI = class UI {
 	 *
 	 * Show the dash and hide some buttons
 	 */
-	toggleDash() {
-		let game = this.game;
 
+	toggleDash: function (randomize) {
```

Comment:
Incorrect definition - all you had to do was add the arg in `toggleDash(randomize) {`